smarts/sstudio/types.py

```diff
@@ -307,11 +307,29 @@ def __eq__(self, other):
         return self.__class__ == other.__class__ and hash(self) == hash(other)
 
 
+@dataclass(frozen=True)
+class JunctionEdgeIDResolver:
```

Comment:
This here is a utility for finding the edge inside an intersection since it is not possible(difficult?) to name it inside `netedit`, and otherwise you would need to determine it from the `map.net.xml` file.